tests/test_display.py

```diff
@@ -610,10 +610,10 @@ def synthesized_met(self, request):
         img, model = request.param.split('-')
         # make the images really small so nothing takes as long
         if img == 'rgb':
-            img = po.load_images(op.join(DATA_DIR, 'color_wheel.jpg'), False)
+            img = po.load_images(op.join(DATA_DIR, 'color_wheel.jpg'), False).to(DEVICE)
             img = img[..., :16, :16]
         else:
-            img = po.load_images(op.join(DATA_DIR, 'nuts.pgm'))
+            img = po.load_images(op.join(DATA_DIR, 'nuts.pgm')).to(DEVICE)
```

Comment:
This is good! I'm pretty sure this means that, for those synthesis tests, synthesis gets run on the GPU and then we test that display works as expected? That's a good test and I think sufficient for testing device in test_display.py -- the various display code should all copy things onto cpu for plotting.

Reply:
Yes, this means that tests in `TestMADDisplay` and `TestMetamerDisplay` are run on GPU when available.